### ISSUE TITLE
Lacrimal punctum number

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10193]
**HPO term**
https://hpo.jax.org/app/browse/term/HP:0032514
https://hpo.jax.org/app/browse/term/HP:0007820
**Suggested revision of textual definition**
Both of these terms make it sound like there is normally only one lacrimal punctum per eye, when in fact there is both a superior and inferior lacrimal punctum in each eye.

**Comment by pnrobinson** [comment]
https://onlinelibrary.wiley.com/doi/full/10.1111/cxo.12231

**Comment by MickeySegal** [comment]
There is normally one superior and one inferior: https://www.allaboutvision.com/eye-care/eye-anatomy/lacrimal-punctum/

The paper that you cite describes "inferior double puncta", suggesting, that if the eye has the normal one superior punctum, a total of 3 puncta in that eye.

**Comment by pnrobinson** [comment]
Thanks for pointing out this error.
I have changed the label to 
Supernumerary lacrimal punctum


I added this commentary
Tears mainly produced by the lacrimal gland travel nasally across the eye toward the punctum, found on the medial margin of each eyelid. This term describes a congenital anomaly chanracterized by the presence of more than one lacrimal punctum in an eyelid. Accessory puncta are typically located on the lower lid medial to the normal punctum with a slit-like appearance.

and I added the synonyms listed in the paper.
